signac/syncutil.py

```diff
@@ -26,6 +30,12 @@ def log_more(msg, *args, **kwargs):
 logger.more = log_more  # type: ignore
 
 
+@deprecated(
+    deprecated_in="1.6.0",
+    removed_in="2.3.0",
```

Comment:
We'll remove this in the next major release, 2.0.0. 👍
```suggestion
    removed_in="2.0.0",
```